vite.config.mts

```diff
@@ -37,6 +44,16 @@ export default defineConfig({
     }
   },
   build: {
-    target: 'ESNext'
+    target: 'ESNext',
```

Comment:
Do we want to build to ESNext? I think it is better to support something like ES6, so older browsers without any updates can still use Aurora. I also have a specific use case, where we want to use an old iPad to control lights during a GEWIS event.

Reply:
I did not look in depth which build versions worked well and which did not. If we want to support older versions, I suggest to look into this in another PR later, as changing this now will require a lot of extra changes in this already convoluted PR.